cfgov/unprocessed/js/modules/multi-select.js

```diff
@@ -0,0 +1,494 @@
+'use strict';
+
+function init() {
```

Comment:
This should be formatted as a class and instantiated as an instance to avoid cross-contamination of state if multiple multi-selects appear on the same page.